### PR TITLE
Don't artificially limit the rendering axes for CVRP environment

### LINKED ISSUE #258: [BUG] plotting out of scale
## Describe the bug

CVRPTW generator scales both time and location coordinates with the same factor (max_time). This notably results in a bad rendering.

## To Reproduce

Initialize the CVRPTWGenerator with specific parameters, ensuring that scale=True and max_time is set to a particular value.​
Generate a set of instances using this generator.​
Examine the spatial distribution of the generated locations.

```python
from rl4co.envs.routing.cvrptw.env import CVRPTWEnv
env = CVRPTWEnv(generator_params={'num_loc':20,'scale':True})
env.render(td_init[0],actions=torch.tensor([0,1]))
```

## System info

rl4co : 0.5.2

## Additional context

The render function could also be updated (the axs limit currently assume you have scaled the problem)

## Reason and Possible fixes

Change this : 

if self.scale:
    durations = durations / self.max_time
    min_times = min_times / self.max_time
    max_times = max_times / self.max_time
    td["depot"] = td["depot"] / self.max_time
    td["locs"] = td["locs"] / self.max_time

To this : 
if self.scale:
    durations = durations / self.max_time
    min_times = min_times / self.max_time
    max_times = max_times / self.max_time
    td["depot"] = td["depot"] / self.max_loc
    td["locs"] = td["locs"] / self.max_loc

## Checklist

- [ ✅] I have checked that there is no similar issue in the repo (**required**)
- [ ✅] I have provided a minimal working example to reproduce the bug (**required**)

### PR DESCRIPTION
**Pull request by ngastzepeda**
## Description

Commented our the two lines of code that set the limits for the x and y axis, respectively.

## Motivation and Context

Assuming all locs would be scaled to [0,1] let to rendering problems for instances that were not in fact scaled to [0,1].

Closes #258 .

## Types of changes

What types of changes does your code introduce? Remove all that do not apply:

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by fedebotu** [comment]
Sounds good!
I am also thinking about a more systematic change:
How about we never impose any artificial limit in the `render` function, but instead always return the `ax`?

For example:

```python
def render(td, ..., return_ax=False):
   # Main function
   # ... 
   if return_ax:
      return ax
```


In this way, users can directly modify the plot style (including setting the `x_lim` and `y_lim`) afterwards:


```python
ax = env.render(td, ..., return_ax=True):
ax.set_xlim(0, 1)
ax.set_ylim(0, 1)
```

WDYT?

**Comment by ngastzepeda** [comment]
> Sounds good! I am also thinking about a more systematic change: How about we never impose any artificial limit in the `render` function, but instead return the `ax`?
> 
> For example:
> 
> ```python
> def render(td, ..., return_ax=False):
>    # Main function
>    # ... 
>    if return_ax:
>       return ax
> ```
> 
> In this way, users can directly modify the plot style (including setting the `x_lim` and `y_lim`) afterwards:
> 
> ```python
> ax = env.render(td, ..., return_ax=True):
> ax.set_xlim(0, 1)
> ax.set_ylim(0, 1)
> ```
> 
> WDYT?
> 
> cc @Estalisium, this should solve the plotting issue!

I have not tested this, but feel free to push to this branch @fedebotu :)

**Comment by fedebotu** [comment]
Plotting bug from @Estalisium fixed, no more artificial restriction on range!
 
Also, now all `render` function calls return an `ax` by default!

This can be used to further modify the plot 

```python
ax = env.render(td[0], actions[0])
ax.set_xlim(-0.05, 1.05)
ax.set_ylim(-0.05, 1.05)
ax.set_xticks([])
ax.set_yticks([])
```

Since everything seems to be working well, I will proceed with merging the PR~
